### ISSUE TITLE
Unable to select "Empty Page Template"

### ISSUE THREAD

**Issue by bradweikel** [understrap/understrap#1242]
When editing a page, I'm able to successfully switch it to use most of the Understrap page templates (Blank, Full Width, Left Sidebar, etc), but whenever I select "Empty Page Template" that change doesn't save properly. When I go back to edit the form it's reverted back to Default Template (which I have set to Full). It never renders with the Empty template.

Steps:
1. Create a page
2. While editing the page, change the template to Blank Page Template or some other template.
3. Save
4. View the page or refresh the edit screen to verify the change was applied. 
5. Repeat the change of templates, but this time choose "Empty Page Template"

Result: this time, the change does not save, and upon refreshing the edit screen the setting has reverted to "Default template"

**Comment by bradweikel** [comment]
My gut was telling me this was some sort of naming collision issue, so just to check I tried renaming the template "Empty2" and renamed the page-templates/empty.php to empty2.php, and the issue went away. So it's a collision of some sort.

Since this could be version specific, I'm running Wordpress 5.6 on PHP 7.3, hosted on Pantheon.

**Comment by IanDelMar** [comment]
I can't confirm this issue. On my local install (WP 5.6, PHP 7.4.9) Both creating a page with the empty page template and switching to the empty page template works.

**Comment by bradweikel** [comment]
I turned off my plugins and brought them back one at a time and traced this back to the LiveCanvas plugin. LiveCanvas is a page builder for Understrap that hijacks the page template, sets it to "empty", and overrides the page's content with its own. But if LiveCanvas is turned off for a page AND the template is set to "empty", they incorrectly assume that you JUST turned it off with that save and they change it back to Default. 

Closing this issue. Will file a bug report with them.
